src/current/_includes/releases/v23.2/v23.2.0-alpha.5.md

```diff
@@ -0,0 +1,24 @@
+## v23.2.0-alpha.5
+
+Release Date: October 30, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-alpha-5-sql-language-changes">SQL language changes</h3>
+
+- Added support for the special `OTHERS` condition in PL/pgSQL exception blocks, which allows matching any error code apart from `query_canceled` and `assert_failure`. Note that Class 40 errors (`40000`, `40001`, `40003`, `40002`, and `40P01`) cannot be caught either. This is tracked in [#111446](https://github.com/cockroachdb/cockroach/pull/111446). [#112817][#112817]
```

Comment:
```suggestion
- Added support for the special `OTHERS` condition in PL/pgSQL exception blocks, which allows matching any error code apart from `query_canceled` and `assert_failure`. Note that Class 40 errors (`40000`, `40001`, `40003`, `40002`, and `40P01`) cannot be caught either. This is tracked in [#111446](https://github.com/cockroachdb/cockroach/pull/111446), [#112817][#112817]
```

Reply:
I think this changes the meaning of the release note. The release note itself resolves #112817. A separate but related issue is tracked in #111446.

Reply:
Ah, I see.